src/wallet/wallet.cpp

```diff
@@ -41,7 +41,7 @@ CAmount maxTxFee = DEFAULT_TRANSACTION_MAXFEE;
 unsigned int nTxConfirmTarget = DEFAULT_TX_CONFIRM_TARGET;
 bool bSpendZeroConfChange = true;
 bool fSendFreeTransactions = false;
-bool fPayAtLeastCustomFee = true;
+bool fPayAtLeastCustomFee = false;
```

Comment:
The tests need this to be `false` but you are right: This PR is not merge ready as of now. This needs at least one follow up commit but I am not sure which path to take:
* Option 1: Make this PR change the default and do the required things (i.e. adjust release notes et al.)
* Option 2: Introduce a command line parameter (or rpc?) to overwrite the default. (I am not sure if we want yet another param to keep track of... )